### ISSUE TITLE
yarn install --frozen-lockfile --non-interactive failed with code 1

### ISSUE THREAD

**Issue by atonamy** [postlight/serverless-babel-starter#64]
Unable to deploy code from my computer
```
Serverless: Invoke webpack:package
Serverless: Fetch dependency graph from package.json
Serverless: WARNING: Could not determine version of module aws-sdk
Serverless: Package lock found - Using locked versions
Serverless: Packing external modules: typeorm@^0.2.17, aws-sdk
 
  Error --------------------------------------------------
 
  yarn install --frozen-lockfile --non-interactive failed with code 1
 
     For debugging logs, run again after setting the "SLS_DEBUG=*" environment variable.
 
  Stack Trace --------------------------------------------
 
Error: yarn install --frozen-lockfile --non-interactive failed with code 1
    at ChildProcess.child.on.exitCode (node_modules/serverless-webpack/lib/utils.js:92:16)
    at ChildProcess.emit (events.js:189:13)
    at ChildProcess.EventEmitter.emit (domain.js:441:20)
    at maybeClose (internal/child_process.js:970:16)
    at Socket.stream.socket.on (internal/child_process.js:389:11)
    at Socket.emit (events.js:189:13)
    at Socket.EventEmitter.emit (domain.js:441:20)
    at Pipe._handle.close (net.js:597:12)
 
  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Issues:        forum.serverless.com
 
  Your Environment Information ---------------------------
     OS:                     darwin
     Node Version:           10.15.3
     Serverless Version:     1.44.1
```

**Comment by orkunkl** [comment]
I am having the same issue. Please can you resolve this?

**Comment by achwin** [comment]
I resolve that with `npm install aws-sdk --save-dev`

**Comment by WajeehZantout** [comment]
This is fixed in #79
